src/pages/public/StagingServerOverview/components/FetchUtils.ts

```diff
@@ -0,0 +1,21 @@
+const serverURL = "https://staging.vektorprogrammet.no/api/servers";
+const diskUsageURL = "https://staging.vektorprogrammet.no/api/disk-space";
+
+export async function fetchServers(): Promise<any> {
```

Comment:
Make use of TypeScript?

```suggestion
export async function fetchServers(): Promise<[StagingServerAPIResponse]> {
```